src/app/app.config.ts

```diff
@@ -1,8 +1,27 @@
-import { ApplicationConfig } from '@angular/core';
+import { ApplicationConfig, importProvidersFrom } from '@angular/core';
+import { initializeApp, provideFirebaseApp } from '@angular/fire/app';
+import { getAuth, provideAuth } from '@angular/fire/auth';
+import { getFirestore, provideFirestore } from '@angular/fire/firestore';
 import { provideRouter } from '@angular/router';
-
+import { FIRESTORE_API_KEY } from '../../environments/firestore-api-key';
 import { routes } from './app.routes';
 
+const firebaseConfig = {
+  apiKey: FIRESTORE_API_KEY.apiKey,
+  authDomain: 'angular-book-tracker-686b9.firebaseapp.com',
+  projectId: 'angular-book-tracker-686b9',
+  storageBucket: 'angular-book-tracker-686b9.appspot.com',
+  messagingSenderId: '706862192768',
+  appId: '1:706862192768:web:cab46e521fb925ede86e65',
+};
+
```

Comment:
You could potentially hide your entire `firebaseConfig` in  `environments`. However, this is just a suggestion.